scripts/jinx.lic

```diff
@@ -533,10 +543,15 @@ module Jinx
       remote_name  = Installer.normalize_filename(remote_name)
       sources = sources.map {|source| Repo.manifest(source) }
       repo = Installer.ensure_specific(remote_name, sources)
-      asset = repo[:available].find {|remote_asset|
-        File.basename(remote_asset[:file]).eql?(remote_name)
-      }
-      local_asset_name = File.basename(local_asset_name || asset[:file])
+      asset = Jinx::Lookup.find_asset_in(repo, remote_name)
+      local_asset_name = URI.decode(File.basename(local_asset_name || asset[:file]))
```

Comment:
Same here for depreciated URI.decode method. This should fix it: `local_asset_name = URI.decode_www_form_component(File.basename(local_asset_name || asset[:file]))`

Reply:
Still need to update this to use CGI.escape as well. Then should be good.

Reply:
Thanks, good catch. I missed this one at first but its updated now too.